middlewares/account.middleware.js

```diff
@@ -83,12 +83,48 @@ async function updatePassword(req, res, next) {
     return next();
 }
 
-// TODO: fix when new permission system is created
-async function addDefaultHackerPermissions(req, res, next) {
```

Comment:
What happened to this function?

Reply:
This was a remnant of the old default permissions system. We have new middleware in place now. I feel like I've deleted this function several times, but it keeps popping up for some reason.